libpam/pam_google_authenticator.c

```diff
@@ -1383,6 +1397,16 @@ static int parse_args(pam_handle_t *pamh, int argc, const char **argv,
       }
       params->fixed_uid = 1;
       params->uid = uid;
+    } else if (!memcmp(argv[i], "allow_perm=", 11)) {
```

Comment:
Just for consistency. The "user=" and "secret=" options use memcmp too, though I can change them all to strncmp